onfido-java/src/test/java/com/onfido/webhooks/WebhookEventVerifierTest.java

```diff
@@ -10,7 +10,7 @@
   WebhookEventVerifier verifier = new WebhookEventVerifier(webhookToken);
 
   String rawEvent =
-      "{\"payload\":{\"resource_type\":\"check\",\"action\":\"check.completed\",\"object\":{\"id\":\"check-123\",\"status\":\"complete\",\"completed_at_iso8601\":\"2020-01-01T00:00:00Z\",\"href\":\"https://api.onfido.com/v3/checks/check-123\"}}}";
+      "{\"payload\":{\"resource_type\":\"check\",\"action\":\"check.completed\",\"object\":{\"id\":\"check-123\",\"status\":\"complete\",\"completed_at_iso8601\":\"2020-01-01T00:00:00Z\",\"href\":\"https://api.onfido.com/v3.1/checks/check-123\"}}}";
```

Comment:
This href and the one below won't be changing - should also fix your failing test too as the signature isn't valid if the contents are changed.